### DISCUSSION TITLE
Daylight saving times issues

### DISCUSSION THREAD

**Discussion by drfrostongithub** [dr5hn/countries-states-cities-database#267] [Ideas]
Just looking and using your countries+states.json, was using Australia, but it still using the old Daylight saving times, when aus didn't use it anymore, care to fix every DST? you can also have dstoffset field = true/false.

Thanks

**Comment by dr5hn** [comment]
@drfrostongithub Thanks for reporting. 🎉 

Yes, sure. Would you please let me know what will be the correct DST values that need to be updated?
Also adding one more field is a nice idea. Thanks

**Comment by drfrostongithub** [comment]
@dr5hn Hi Dude, 

You can look DST complete list here https://en.wikipedia.org/wiki/Daylight_saving_time_by_country

Hope this help your project.
